src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -258,15 +275,15 @@ export default class GitOpsConfiguration extends Component<GitOpsProps, GitOpsSt
                             onChange={(event) => this.handleChange(event, 'username')}
                             name="Enter username" error={this.state.isError.username}
                             tabIndex={3}
-                            label={this.state.tab === GitProvider.Github ? "GithHub Username*" : "GitLab Username*"}
+                            label={this.state.tab === GitProvider.Github ? "GithHub Username*" : this.state.tab === GitProvider.GitLab ? "GitLab Username*" : "Azure DevOps Username*"}
```

Comment:
earlier we had gitLabGroupId as default over here we have azure as default. we should have consistent behaviour

Reply:
fixed